webapp/i18n/ja.json

```diff
@@ -2012,6 +2054,7 @@
   "user.settings.security.ldap": "AD/LDAP",
   "user.settings.security.loginGitlab": "GitLabでログインしました",
   "user.settings.security.loginLdap": "AD/LDAPでログインする",
+  "user.settings.security.loginSaml": "GitLabでログインしました",
```

Comment:
Thanks @kaakaa for the reply

"Fuzzy" translations are marked as "Needs work" in the Translation Server, and they require approval from someone before a language can get back to 100%.

![image](https://cloud.githubusercontent.com/assets/13119842/20641152/f37d9a52-b3bf-11e6-8c4f-c5354c3c5597.png)